src/labels/CustomLabels.labels

```diff
@@ -6627,7 +6684,7 @@ Selecting Don't Auto Schedule Default NPSP Jobs tells NPSP not to check for or r
         <language>en_US</language>
         <protected>false</protected>
         <shortDescription>stgHelpAdvancedMapping3</shortDescription>
-        <value>Find complete Advanced Mapping setup documentation &lt;a href=&quot;https://powerofus.force.com/&quot; target=&quot;_blank&quot;&gt;here&lt;/a&gt;.</value>
+        <value>Find complete Advanced Mapping setup documentation &lt;a href=&quot;https://powerofus.force.com/NPSP_Advanced_Mapping&quot; target=&quot;_blank&quot;&gt;here&lt;/a&gt;.</value>
```

Comment:
@melissabarber - with the conversation with EntSys, is this what we want to have in the label?

Reply:
Yep, to this one too.